ibtk/src/lagrangian/FEDataManager.cpp

```diff
@@ -3025,6 +3050,8 @@ FEDataManager::collectActivePatchElements(std::vector<std::vector<Elem*> >& acti
     const MeshBase& mesh = d_fe_data->d_es->get_mesh();
     System& X_system = d_fe_data->d_es->get_system(COORDINATES_SYSTEM_NAME);
 
+    // We may
```

Comment:
??

Reply:
I was going to write a comment here but it ended up not being necessary. Let me delete the stub.